inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/utils/ProxyUtils.java

```diff
@@ -104,11 +104,19 @@ public static String getJarVersion() {
         if (sdkVersion != null) {
             return sdkVersion;
         }
-        Properties properties = new Properties();
-        try (InputStream is = ProxyUtils.class.getResourceAsStream("/git.properties")) {
-            properties.load(is);
-            sdkVersion = properties.getProperty("git.build.version");
+        try (InputStream is = ProxyUtils.class.getClassLoader().getResourceAsStream("sdk.version")) {
```

Comment:
<resources>
			<resource>
				<directory>src/main/resources</directory>
				<filtering>false</filtering>
			</resource>
			<resource>
				<directory>src/main/resources-filtered</directory>
				<filtering>true</filtering>
			</resource>
		</resources>
		
		
https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html

Using Maven resource filtering seems to be a good solution, which does not rely on git.